### PR TITLE
adding guacanalyze to guacone

### PR DESCRIPTION
**Pull request by arorasoham9**
# Description of the PR

Adding analyze command to guacone

# PR Checklist

- [ ] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [ ] All new changes are covered by tests
- [ ] If GraphQL schema is changed, `make generate` has been run
- [ ] If OpenAPI spec is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [ ] All CI checks are passing (tests and formatting)
- [ ] All dependent PRs have already been merged

### CODE REVIEW

**Comment by nathannaveen** [comment]
@arorasoham9, are you ready for a review?

**Comment by arorasoham9** [comment]
@nathannaveen I think I'll add some more commits to this.

**Comment by pxp928** [comment]
> @nathannaveen I think I'll add some more commits to this.

Could you make this a draft PR until it is ready for review?

**Comment by pxp928** [comment]
@arorasoham9 please mark it for review when its ready

File: pkg/guacanalyze/root.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	"strings"
+
+	"github.com/guacsec/guac/pkg/cli"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	cobra.OnInitialize(cli.InitConfig)
```
Comment (pxp928) [inline pkg/guacanalyze/root.go:30]: keep the CLI in https://github.com/guacsec/guac/tree/main/cmd/guacone/cmd

Only move the other logic to this package.

**Comment by pxp928** [comment]
Can you also fix the lint and static analysis errors?

**Pull request closed without merging**